tasks/tiler/xyz.py

```diff
@@ -187,22 +177,3 @@ def output(self):
             table_schema = self._get_table_schema(table_config)
             targets.append(PostgresTarget(table_schema['schema'], table_schema['table_name'], where='z = {}'.format(self.zoom_level)))
         return targets
-
-
-class AllXYZTables(WrapperTask):
```

Comment:
This is mentioned in the Makefile, it should change.

Reply:
do.py is removed, now is called xyz

Reply:
changed in the makefile